src/components/forms/CloudInitForm.tsx

```diff
@@ -29,12 +34,39 @@ export const cloudInitPayload = (values: InstanceAndProfileFormValues) => {
 
 interface Props {
   formik: InstanceAndProfileFormikProps;
+  project: string;
 }
 
-const CloudInitForm: FC<Props> = ({ formik }) => {
-  const getCloudInitRow = (label: string, name: string, value?: string) => {
-    const metadata = getConfigRowMetadata(formik.values, name);
```

Comment:
Can we keep the `metadata = getConfigRowMetadata` inside the `getCloudInitRow` function? Seems too complex to pull it out and define multiple variables for it just to pass it back in again.